tools/shared/src/main/scala/org/scalajs/core/tools/classpath/IRClasspath.scala

```diff
@@ -42,8 +43,8 @@ final class IRClasspath(
     if (scalaJSIR.nonEmpty) {
       val semantics = Semantics.compliantTo(requiredCompliance.map(_.semantics))
       val output = WritableMemVirtualJSFile(outName)
-      new ScalaJSOptimizer(semantics).optimizeCP(this,
-          Config(output), NullLogger)
+      new ScalaJSOptimizer(semantics, OutputMode.ECMAScript51Global).optimizeCP(
+          this, Config(output), NullLogger)
```

Comment:
I do not think the specification of the interface of `Classpath` gives any guarantee about the content of the file. We can explicitate that if you like.